modules/plugin/shapefile/src/main/java/org/geotools/data/shapefile/ShapefileFeatureSource.java

```diff
@@ -502,6 +502,7 @@ protected SimpleFeatureType buildFeatureType() throws IOException {
                     crs = prj.getCoordinateReferenceSystem();
                 }
             } catch (FactoryException fe) {
+                LOGGER.log(Level.FINE, "Error reading prj file", fe);
```

Comment:
We have several users setting logging to "VERBOSE" and then complaining there are "errors" in the logs just because they see stack traces. Could you reword the message to at least state this is likely not fatal and could actually even be expected? (same for the other logs added in the PR).